packages/mobile/src/components/details-tile/DetailsTileHasAccess.tsx

```diff
@@ -231,61 +243,38 @@ export const DetailsTileHasAccess = ({
       )
     }
     if (followee) {
-      return (
-        <View style={styles.descriptionContainer}>
-          <Text>
-            <Text style={styles.description}>
-              {messages.unlockedFollowGatedPrefix}
-            </Text>
-            <Text style={styles.description}>{followee.name}</Text>
-            <UserBadges
-              badgeSize={16}
-              user={followee}
-              nameStyle={styles.description}
-              hideName
-            />
-            <Text style={styles.description}>
-              {messages.unlockedFollowGatedSuffix}
-            </Text>
-          </Text>
-        </View>
-      )
+      return renderUnlockedSpecialAccessDescription({
+        entity: followee,
+        prefix: messages.unlockedFollowGatedPrefix,
+        suffix: messages.unlockedFollowGatedSuffix
+      })
     }
     if (tippedUser) {
-      return (
-        <View style={styles.descriptionContainer}>
-          <Text>
-            <Text style={styles.description}>
-              {messages.unlockedTipGatedPrefix}
-            </Text>
-            <Text style={styles.description}>{tippedUser.name}</Text>
-            <UserBadges
-              badgeSize={16}
-              user={tippedUser}
-              nameStyle={styles.description}
-              hideName
-            />
-            <Text style={styles.description}>
-              {messages.unlockedTipGatedSuffix}
-            </Text>
-          </Text>
-        </View>
-      )
+      return renderUnlockedSpecialAccessDescription({
+        entity: tippedUser,
+        prefix: messages.unlockedTipGatedPrefix,
+        suffix: messages.unlockedTipGatedSuffix
+      })
     }
 
     console.warn(
       'No entity for premium conditions... should not have reached here.'
     )
     return null
-  }, [nftCollection, followee, tippedUser, handlePressCollection, styles])
+  }, [
+    nftCollection,
+    followee,
+    tippedUser,
+    handlePressCollection,
+    renderUnlockedSpecialAccessDescription,
+    styles
+  ])
 
   if (isOwner) {
     return (
       <DetailsTileOwnerSection
-        nftCollection={nftCollection}
-        followee={followee}
-        tippedUser={tippedUser}
         renderDescription={renderOwnerDescription}
+        isCollectibleGated={!!nftCollection}
```

Comment:
I like the simplification of this logic into a single variable! Just want to make sure we don't have any need for those individual properties in the child component?

Reply:
yeah we should be good